enterprise-suite/console_test/smoke_ingress_test.go

```diff
@@ -0,0 +1,87 @@
+package console
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	extv1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/lightbend/console_test/args"
+
+	"github.com/lightbend/console_test/util/minikube"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// Console NodePort service
+const consoleServiceName = "expose-es-console"
+
+// Name of Ingress that this test creates
+const testIngressName = "console-test-ingress"
+
+// TODO(mitkus): Make this test work. On my machine both this and bash e2e smoke_ingress
+// stopped working after minikube update.
+var _ = XDescribe("minikube:ingress", func() {
```

Comment:
I guess this is WIP then?

Reply:
I'll tackle this next after merging what's currently working